apps/02RockPaperScissors.js

```diff
@@ -11,7 +11,43 @@ var rl = readline.createInterface({
 function rockPaperScissors(hand1, hand2) {
 
   // Write code here
```

Comment:
You can shorten your code if you added this after the function is declared:
name1 = name1.toLowerCase();
name2 = name2.toLowerCase();